elastic/kibana
# feat(slo): prevent initial backfill

Resolves https://github.com/elastic/kibana/issues/181633

## Summary

This PR adds an optional settings: `preventInitialBackfill`. The Create/Edit form show this settings under the objective section.
When this settings is checked, we start the transform now minus some delay buffer. 

### How to test

- Create some SLO with or without checking this settings: when true, no historical data should be computed.
- Edit SLOs: make sure the settings is correctly pre-populated with initial value
- Clone SLOs: make sure the settings is correctly pre-populated with initial value
